packages/db/src/AsyncOperations.ts

```diff
@@ -0,0 +1,10 @@
+import Knex from 'knex';
+
+import { tableNames } from './tables';
+import { doesRecordExist } from './database';
+import { AsyncOperationRecord } from './types';
+
+export const doesAsyncOperationExist = async (
```

Comment:
Nit: these should probably have thin units.    It's fine if we add them later in the ticket too. 

----
Non blocking discussion: 

These wrappers seem *really* thin, but getting the typing *does* seem totally worth it:

```
import { doesAsyncOperationExist, doesFooOperationExist } from '@cumulus/db'
doesAsyncOperationExist(params, knex) 
doesFooOperationExist(params, knex) 
```

over 

```
import { doesRecordExist, tableNames } from '@cumulus/db'
doesRecordExist(params, knex, tableNames.async)
doesRecordExist(params, knex, tableNames.foo)
```

Maybe to file away for another day, but I feel like we should consider if there's a way to determine the return type based on the tableName.......... that's probably complex over explicit though.